### PR TITLE
update expired links

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

+ Updated expired links in the docs

### CODE REVIEW

File: docs/guide/execute-transaction.md [hunk 1]
```diff
@@ -123,7 +124,7 @@ To deploy an ASA without using `asa.yaml`:
 
 - To learn about more parameters like (account, appArgs, ForeignApps, ForeignAssets etc).Please check [AppOptionalFlags](https://algobuilder.dev/api/algob/interfaces/runtime.types.AppOptionalFlags.html)
 
-### [Opt-In to App](https://algobuilder.dev/api/algob/modules/runtime.types.html#optintoappparam)
+### [Opt-In to App](https://algobuilder.dev/api/web/enums/types.TransactionType.html#OptInToApp)
```
Comment (sczembor) [inline docs/guide/execute-transaction.md:127]:
```suggestion
### [Opt-In to App](https://algobuilder.dev/api/runtime/classes/Runtime.html#optInToApp)
```

File: docs/guide/debugging-teal.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ If it displays the `tealdbg` help, you're good to go.
 
 ## Using TEAL debugger with algob
 
-Creating transaction data (via `goal --dryrun-dump` or SDK) could be a lengthy process, especially when using a transaction group. `Algob` provides an easy way to use debugger: by simply supplying the transactions as an input to the `TealDbg` method (same transaction parameters that we supply to [executeTx](https://algobuilder.dev/api/algob/modules.html#executetransaction) to execute same transaction on network).
+Creating transaction data (via `goal --dryrun-dump` or SDK) could be a lengthy process, especially when using a transaction group. `Algob` provides an easy way to use debugger: by simply supplying the transactions as an input to the `TealDbg` method (same transaction parameters that we supply to [executeTx](https://algobuilder.dev/api/algob/modules.html#executeTransaction) to execute same transaction on network).
```
Comment (vuvth) [inline docs/guide/debugging-teal.md:19]: We renamed executeTransaction to executeTx. Let create TODO for it. We can update link after released new version.

Comment (vuvth) [inline docs/guide/debugging-teal.md:19]: We renamed executeTransaction to executeTx. Let create TODO for it. We can update link after released new version.

File: docs/guide/deployer.md [hunk 3]
```diff
@@ -116,9 +116,8 @@ Smart contracts must be stored in `assets` folder.
 
 The main difference between deploying an ASA and App is that ASA takes `asset-name` and `ASADeploymentFlags` as input and App takes `smart-contract-names` and `AppDeploymentFlags` as input.
 
-You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/api/algob/interfaces/types.deployer.html);
+You can learn more about the flags from [Deployer API](https://algobuilder.dev/api/algob/interfaces/types.Deployer.html);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
```
Comment (vuvth) [inline docs/guide/deployer.md:120]:
```suggestion
You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/get-details/dapps/smart-contracts/apps/).
```

**Review by vuvth — APPROVED** [review]
LGTM! Let apply review suggestion.

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**